processing/data_collection/gazette/spiders/df_brasilia.py

```diff
@@ -0,0 +1,85 @@
+import re
+import json
+import itertools
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class DfBrasiliaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "5300108"
+    GAZETTE_URL = "http://dodf.df.gov.br/listar"
+
+    MONTHS = {
+        "12": "12_Dezembro",
+        "11": "11_Novembro",
+        "10": "10_Outubro",
+        "09": "09_Setembro",
+        "08": "08_Agosto",
+        "07": "07_Julho",
+        "06": "06_Junho",
+        "05": "05_Maio",
+        "04": "04_Abril",
+        "03": "03_Março",
+        "02": "02_Fevereiro",
+        "01": "01_Janeiro",
+    }
+    YEARS = ["2018", "2017", "2016", "2015"]
```

Comment:
You are hard-coding the years you are collecting the data. This spider will stop collecting data in 2019. You also don't need to set a lower limit for the spider as we want to get as much data as possible (and if we need to limit), there is already a pipeline that drops the unwanted item. Remove this list and implement the crawling in the initial page to identify which years are available.